uiowa/uiowa
# [Registrar] Course deadlines

Resolves #7885 




# How to test


```
ddev blt ds --site=registrar.uiowa.edu && ddev drush @registrar.local uli
```

- Edit a page's layout and add a Course Deadlines block, save
- Use the deadlines block and compare the (unthemed) results with https://registrar.uiowa.edu/course-deadlines
- Check for an off-cycle course like 'Summer 2024, ABRD, 3016, 0001' or others, and see that it has an "this is an off-cycle course" label
- Try different combinations of filling out the fields then changing one of the previous fields--results and fields below the changed one should reset.
- When choosing a course with only one available section, that section should autopopulate the section field and the deadlines info block should appear
